### PR TITLE
doc improvement: moved testing section from nrf52

### PR DESCRIPTION
**Pull request by divipillai**
Moved testing section from nrf52 to Peripheral UART sample
as per the comment here: https://github.com/nrfconnect/sdk-nrf/pull/15598#discussion_r1628848604

### CODE REVIEW

File: samples/bluetooth/peripheral_uart/README.rst [hunk 1]
```diff
@@ -205,6 +205,43 @@ After programming the sample to your development kit, complete the following ste
...
+
+Testing with multiple nRF52 Series devices
+------------------------------------------
+
+If you have an nRF52 Series DK with the :ref:`peripheral_uart` sample and either a dongle or second Nordic Semiconductor development kit that supports Bluetooth Low Energy, you can test the sample on your computer.
+Use the Bluetooth Low Energy app in `nRF Connect for Desktop`_ for testing.
+
+To perform the test, :ref:`connect to the nRF52 Series DK <nrf52_gs_connecting>` and complete the following steps:
+
+1. Connect the dongle or second development kit to a USB port of your computer.
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_uart/README.rst:218]:
```suggestion
To perform the test, complete the following steps:

1. Connect to your nRF52 Series DK. 
#. Connect the dongle or second development kit to a USB port of your computer.
```
(connecting is already mentioned at the start of Testing, no need to point to GS here)

File: samples/bluetooth/peripheral_uart/README.rst [hunk 2]
```diff
@@ -205,6 +205,43 @@ After programming the sample to your development kit, complete the following ste
...
+#. Open the Bluetooth Low Energy app.
+#. Select the serial port that corresponds to the dongle or second development kit.
+   Do not select the kit you want to test.
+
+   .. note::
+      If the dongle or second development kit has not been used with the Bluetooth Low Energy app before, you may be asked to update the J-Link firmware and connectivity firmware on the nRF SoC to continue.
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_uart/README.rst:224]:
```suggestion
      If the dongle or the second development kit has not been used with the Bluetooth Low Energy app before, you may be asked to update the J-Link firmware and connectivity firmware on the nRF SoC to continue.
```

File: samples/bluetooth/peripheral_uart/README.rst [hunk 3]
```diff
@@ -205,6 +205,43 @@ After programming the sample to your development kit, complete the following ste
...
+
+If you have an nRF52 Series DK with the :ref:`peripheral_uart` sample and either a dongle or second Nordic Semiconductor development kit that supports Bluetooth Low Energy, you can test the sample on your computer.
+Use the Bluetooth Low Energy app in `nRF Connect for Desktop`_ for testing.
+
+To perform the test, :ref:`connect to the nRF52 Series DK <nrf52_gs_connecting>` and complete the following steps:
+
+1. Connect the dongle or second development kit to a USB port of your computer.
+#. Open the Bluetooth Low Energy app.
+#. Select the serial port that corresponds to the dongle or second development kit.
+   Do not select the kit you want to test.
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_uart/README.rst:221]:
```suggestion
   Do not select the kit you want to test just yet.
```

File: samples/bluetooth/peripheral_uart/README.rst [hunk 4]
```diff
@@ -262,6 +262,44 @@ After programming the sample to your development kit, complete the following ste
       #. Disconnect the device in nRF Connect for Mobile.
          Observe that **LED 1** turns off.
 
+.. _nrf52_computer_testing:
+
+Testing with multiple nRF52 Series devices
```
Comment (grochu) [inline samples/bluetooth/peripheral_uart/README.rst:267]: This title could be more generic as nRF Connect for Desktop can run also with other devices e.g. nRF5340 DK. How about "_Testing with nRF Connect for Desktop_"?

**Review by grochu — COMMENTED** [review]
Looks good, just leaving a suggestion for change

**Review by grochu — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**